source/extensions/tracers/opentelemetry/samplers/sampler.h

```diff
@@ -29,11 +29,21 @@ enum class Decision {
   RECORD_AND_SAMPLE
 };
 
+/**
+ * @brief A string key-value map that stores the span attributes.
+ */
+using OTelSpanAttributes = std::map<std::string, std::string>;
+
+/**
+ * @brief The type of the span.
+ */
+using OtelSpanKind = ::opentelemetry::proto::trace::v1::Span::SpanKind;
```

Comment:
```suggestion
using OTelSpanKind = ::opentelemetry::proto::trace::v1::Span::SpanKind;
```

But not sure the linters will like the two upper "OT" here..